### DISCUSSION TITLE
Can't create categories on Chamilo 1.11.26 (PHP error)

### DISCUSSION THREAD

**Discussion by a-mian-lateral** [chamilo/chamilo-lms#5163] [General]
Hi, sorry to bother you again, but I'm still trying to get familiar with Chamilo and I'm facing another problem.

I have 3 installations, two in a vps with plesk and another in my local machine, in my local machine and in one of the installations in plesk I cannot create a category because of this error:

`Fatal error: Uncaught Error: Call to undefined function html_filter() in /var/www/html/public/main/inc/lib/course_category.lib.php:199`

Digging into the code, I discovered that the php file in which the function is defined is not included (I checked with `get_included_files()`).

If I manually include the file right before the line with the error then the creation process ends up fine
`require_once __DIR__ . "/formvalidator/FormValidator.class.php";`

I am pretty sure there are no differences between the three instances, I mean the code and the production environment are always the same.

**Comment by jgnosolored** [comment]
Hi @a-mian-lateral 

It is advisable to search Chamilo's Github repository (html_filter) and check if there are commits with the same topic

https://github.com/chamilo/chamilo-lms/issues/4916 - Course categories creation breaks

Check here:

https://github.com/chamilo/chamilo-lms/pull/4927/commits/48f192df355262034ef186870b1d53a47e5dfdc5

It was resolved. You can apply the patch to your installation by modifying the line in the main/inc/global.inc.php file

**Reply by a-mian-lateral** [reply]
I'll be more careful next time before creating a new discussion. Thanks for your support
